### PR TITLE
Replace Metrics v2 protocol with MQE in UI templates and E2E Test

### PR DESCRIPTION
**Pull request by mrproliu**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
@mrproliu Could you confirm, after this PR, that the OpenFunction templates are the only ones we used for v2 metric only?
We should be safe to remove the legacy UI metric setup feature from UI side. Right? 

cc @Fine0830

**Comment by mrproliu** [comment]
@Fine0830 Do we have a way to distinguish which ones are using the metrics v2 version? I cannot confirm with 100% certainty right now.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/browser/browser-app.json [hunk 1]
```diff
@@ -264,7 +264,7 @@
                   "h": 14,
                   "i": "14",
                   "type": "Widget",
-                  "metrics": [
+                  "expressions": [
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/browser/browser-app.json:267]: I think this is the way to separate v2 and v3.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/general/general-instance.json [hunk 2]
```diff
@@ -868,7 +862,14 @@
                   },
                   "widget": {
                     "title": "Instance CPU Usage (%)"
-                  }
+                  },
+                  "metricMode":"Expression",
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/general/general-instance.json:866]: I think this is the way to separate v2 and v3.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/mesh/mesh-instance.json [hunk 3]
```diff
@@ -217,21 +218,22 @@
...
                     "showYAxis": true
                   },
-                  "metrics": [
-                    "envoy_total_connections_used",
-                    "envoy_parent_connections_used"
-                  ],
-                  "metricTypes": [
-                    "readMetricsValues",
-                    "readMetricsValues"
-                  ],
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/mesh/mesh-instance.json:227]: I think this is the way to separate v2 and v3.

**Comment by Fine0830** [comment]
> @Fine0830 Do we have a way to distinguish which ones are using the metrics v2 version? I cannot confirm with 100% certainty right now.

Currently, only metric v2 is available in the UI.

**Comment by wu-sheng** [comment]
> > @Fine0830 Do we have a way to distinguish which ones are using the metrics v2 version? I cannot confirm with 100% certainty right now.
> 
> Currently, only metric v2 is available in the UI.

I think @mrproliu 's point is, how to know which is using MQE, or v2 query. MQE is v3 protocol.

**Comment by Fine0830** [comment]
> > > @Fine0830 Do we have a way to distinguish which ones are using the metrics v2 version? I cannot confirm with 100% certainty right now.
> > 
> > 
> > Currently, only metric v2 is available in the UI.
> 
> I think @mrproliu 's point is, how to know which is using MQE, or v2 query. MQE is v3 protocol.

The `metricMode` value is `Expression`, which is using MQE. And The `metricMode` value is `General`, which is using v2 query in the templates.

**Comment by mrproliu** [comment]
> > > > @Fine0830 Do we have a way to distinguish which ones are using the metrics v2 version? I cannot confirm with 100% certainty right now.
> > > 
> > > 
> > > Currently, only metric v2 is available in the UI.
> > 
> > 
> > I think @mrproliu 's point is, how to know which is using MQE, or v2 query. MQE is v3 protocol.
> 
> The `metricMode` value is `Expression`, which is using MQE. And The `metricMode` value is `General`, which is using v2 query in the templates.

Ok, based on this condition, I believe that all UI templates except those starting with "fass-" have been replaced with metrics v3 protocol.

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
@mrproliu Could you update the showcase and redeploy demo? Then we could check whether all graphs work.

**Comment by mrproliu** [comment]
Got it.

**Comment by wu-sheng** [comment]
@Fine0830 Could you prepare a pull request to remove `General` metric setup for all widget components? And verify the UI after @mrproliu redeployment? Notice, the Function part should be excluded from the test. It is either being fixed later or removed.

**Pull request merged**